components/compositing/lib.rs

```diff
@@ -7,145 +7,43 @@
 #[macro_use]
 extern crate log;
 
-pub use crate::compositor::CompositingReason;
 pub use crate::compositor::IOCompositor;
 pub use crate::compositor::ShutdownState;
-pub use crate::compositor_thread::CompositorProxy;
-use embedder_traits::Cursor;
-use gfx_traits::Epoch;
-use ipc_channel::ipc::IpcSender;
-use keyboard_types::KeyboardEvent;
-use msg::constellation_msg::PipelineId;
-use msg::constellation_msg::TopLevelBrowsingContextId;
-use msg::constellation_msg::{BrowsingContextId, TraversalDirection};
-use script_traits::{
-    AnimationTickType, LogEntry, WebDriverCommandMsg, WindowSizeData, WindowSizeType,
-};
-use script_traits::{
-    CompositorEvent, ConstellationControlMsg, LayoutControlMsg, MediaSessionActionType,
-};
-use servo_url::ServoUrl;
-use std::collections::HashMap;
-use std::fmt;
-use std::time::Duration;
+
+use compositing_traits::CompositorProxy;
+use compositing_traits::CompositorReceiver;
+use compositing_traits::ConstellationMsg;
```

Comment:
Since these lines are changing, do you mind using the more modern style of having these on the same line?

Reply:
done in 96eec79caf5fb6dbfab8adaea382b78ca2bb8076